### PR TITLE
Feature/LoFTR_image_local_feature_matching

### PR DESCRIPTION
**Pull request by ethnhe**
Add image local feature matching LoFTR pipeline and model.

### CODE REVIEW

**Comment by wenmengzhou** [comment]
merge latest master and rerun the tests

**Review by wenmengzhou — CHANGES_REQUESTED** [review]
images in test/images  should be uploaded using git-lfs, please refer to

https://github.com/modelscope/modelscope/blob/master/docs/source/develop_cn.md#23-%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%E5%AD%98%E5%82%A8

File: modelscope/pipelines/cv/image_local_feature_matching_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.image_local_feature_matching,
+    module_name=Pipelines.image_local_feature_matching)
+class ImageLocalFeatureMatchingPipeline(Pipeline):
+    """ Image Local Feature Matching Pipeline.
```
Comment (wenmengzhou) [inline modelscope/pipelines/cv/image_local_feature_matching_pipeline.py:24]: add pipeline doc str, refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L34C1-L34C1

**Comment by ethnhe** [comment]
Modified as mentioned.

**Comment by ethnhe** [comment]
Modified as mentioned.

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**